content/en/load-testing/Manual/900-create-report-cheatsheet.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Create Report Cheat-Sheet"
+linkTitle: "Cheat-Sheet Create_report"
+
+weight: 900
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available command options below.
+---
+## Usage
+- `create_report [<options>] <inputdir>` is used to generate a test report.
+- `create_report --help` displays all the available command options and a brief guide about their usage.
+
+### Example(Windows PowerShell)
+
+- A report can be generated in the following way, without using any options.
+```powershell 
+C:\xlt-5.7.1\bin> ./create_report.cmd ../results/20991231-112110 
+```
+## List of Command Options
+- `-ae --exclude-agents <agents>` can be used for excluding a list of comma-separated agents.
+
+- `-ai --include-agents <agents>` is to be used for [including](../540-report-options/#report-for-a-subset-of-agents) a list of comma-separated agents.
+
+-  `-D <property=value>` will override a property from file.
+
+- `-e,--exclude-testcases <test cases>` [excludes](../540-report-options/#excluding-test-scenarios) a comma-separated list of test cases.
+
+- `-from <time>` can be used to ignore results generated before the given time. it can also be combined with `-l` and `-to` for [defining](../540-report-options/#defining-a-reporting-timeframe) a reporting timeframe.
+
+- `-i,--include-testcases <test cases>` can be used to include a list of comma-separated test cases.
+
+- `-l <duration>` is used for utilizing results generated in the specified duration, must be used with `from` or `to` option.
+
+- `-linkToResults <yes|no>` will allow whether or not to link to the result browsers.
```

Comment:
-allow +configure?

Reply:
controls! (as used in the release notes https://xltdoc.xceptance.com/release-notes/4.3.x/#report-generator-1)